### PR TITLE
FEAT : Social Media Screens

### PR DESCRIPTION
**Pull request by MiheerTamkhane**
- Register page
- Home
- Bookmark
- User Profile
- Explore
- Filter

### CODE REVIEW

**Review by narharikale — APPROVED** [review]
great ui mihir🤩
good to go ✨

File: src/frontend/components/feed/Feed.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { Post } from "../post/Post";
+const Feed = () => {
+  return (
+    <div className="bg-gray-900">
+      <CreatePost />
+      <div className="mt-4 gap-4 flex flex-col">
+        <Post />
+        <Post />
+        <Post />
+        <Post />
```
Comment (mimansha-swarup) [inline src/frontend/components/feed/Feed.js:12]: i guess this just for time being,  right?

File: src/frontend/components/navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+export const Navbar = () => {
+  const [expanded, setExpanded] = useState(false);
+  return (
+    <nav className="bg-white border-gray-200 px-2 sm:px-4 py-2.5 dark:bg-purple-700 font-['rajdhani']">
+      <div className="container flex flex-wrap justify-between items-center mx-auto">
+        <NavLink to="/" className="flex items-center">
+          {/* <img
+            src="/docs/images/logo.svg"
+            className="mr-3 h-6 sm:h-9"
```
Comment (mimansha-swarup) [inline src/frontend/components/navbar/Navbar.js]: remove unrequired comments

File: tailwind.config.js [hunk 3]
```diff
@@ -1,7 +1,24 @@
...
+    screens: {
+      sm: { min: "640px" },
+      md: { min: "768px" },
+      lg: { min: "1024px" },
+      xl: { min: "1280px" },
+      minmd: { min: "968px" },
+      maxmidmd: { max: "768px" },
+      maxmd: { max: "968px" },
+      maxxlmd: { max: "1280px" },
+      // => @media (max-width: 767px) { ... }
```
Comment (mimansha-swarup) [inline tailwind.config.js]: remove this comment if not required

**Review by mimansha-swarup — APPROVED** [review]
code looks clean and good folder structure

File: src/frontend/components/feed/Feed.js (hunk 1 above)
Comment (MiheerTamkhane) [reply src/frontend/components/feed/Feed.js:12]: Yupp

File: src/frontend/components/navbar/Navbar.js (hunk 2 above)
Comment (MiheerTamkhane) [reply src/frontend/components/navbar/Navbar.js]: yup I'll Thanks

**Pull request merged**